lua/entities/wtib_chemicalplant_medium/shared.lua

```diff
@@ -5,9 +5,6 @@ ENT.Author			= "kialtia/WarriorXK"
 ENT.Contact			= ""
 ENT.Purpose			= "Uses 50 energy to convert 150 Raw Tiberium into 100 units of poisonous chemical gasses"
 ENT.Instructions	= "Link the Chemical Plant to the required resources and turn it on"
-ENT.Spawnable		= true
-ENT.AdminSpawnable	= true
-ENT.Category		= "Tiberium"

```

Comment:
i think they didn't follow the limits when spawned from the entities tab but the tool spawning them works better. I can add them back if you want.